src/locals/hu.json

```diff
@@ -0,0 +1,145 @@
+    "magnification_of_the_negative": "A Negatívumok felerősítése",
+    "magnification_of_the_negative_explanation": "Ha figyelmen kívül hagyod a sikereket, akkor lehet, hogy \"rontóc\" vagy. Hogyha folyton az elbaltázott interjúkon rágódsz, akkor lehet, hogy nem jut eszedbe, hogy mennyi új tudásra tettél szert általuk.",
+    "magnification_of_the_negative_thought": "Figyeltem a kajálásra a héten, de kihagytam a futást.",
+    "mind_reading": "Gondolatolvasás",
+    "mind_reading_explanation": "Ha azon rágódunk, hogy mások mit gondolnak rólunk, akkor valószínűleg gondolatot szeretnénk olvasni. Ameddig nem mondják meg, hogy mit gondolnak rólunk, addig nem tudhatjuk, hogy tényleg az-e a helyzet, amit mi feltételeztünk. Miért is kéne mindig a legrosszabbra felkészülni?",
+    "mind_reading_thought": "Bunkó voltam Gyurival, mert nem mentem el a bulijára, most biztos utál engem.",
+    "minimization_of_the_positive": "A Sikerek elhallgatása",
+    "minimization_of_the_positive_explanation": "Ha lebecsüljük a sikereinket, akkor minimalizáljuk a (dopamint-tüzelést) [ez hogy mondjuk magyarul? ¿Öröm?]. Akkor is, ha az első napunk nem ment 100%-ig a tervnek megfelelően, nem jelenti azt, hogy pl. 60% ami jól ment azt ignorálhatjuk.",
+    "minimization_of_the_positive_thought": "Sok embernek tetszett az előadásom, de elbaltáztam a bevezetést, szóval biztos elcsettintettem.",
+    "other_blaming": "Mások hibáztatása",
+    "other_blaming_explanation_1": "Ha egy rossz helyzetet mindig valaki másnak a hibájának gondolunk, akkor másokat hibáztatunk. Ha megbuktál egy vizsgán és a tanárt hibáztatod, akkor rossz irányba feccölöd a megoldásra szánt energiáidat. Valaki eléd vágott, az autópályán? Ha rádudálsz, rátapadsz, az segít? Most közveszélyes és ideges is vagy egyszerre!",
+    "other_blaming_explanation_2": "Ez nem jelenti azt, hogy minden negatívumért csak magadat kell okolnod. Nem kell senkit sem hibáztatni. Senkinek sem kell hibáznia ahhoz, hogy hiba történjen. Engedd el, hibáztatás nélkül.",
+    "other_blaming_thought": "Ez a genya előttem túl sokat vacakol a sorban!",
+    "over_generalization": "Túl-álltalánosítás",
+    "over_generalization_explanation": "Ha csak egy példa alapján messzemenő következtetéseket vonunk le, akkor túl-állatlánosítunk. Ha elbaltáztad a prezidet és emiatt azt gondolod, hogy \"rossz\" vagy az előadásban, akkor te túl-álltalánosítasz.",
+    "over_generalization_thought": "Senki sem kért-fel táncolni, szal' biztos nem is fog.",
+    "self_blaming": "Önvád",
+    "self_blaming_explanation": "Ha csak magadat okolod egy kialakult szituációért, az az önvád. Nem kell minden rossz dologért felelősnek lenned. Ha a dugóban ragadsz és azon őrlődsz, hogy miért nem indultál el korábban, az önvád. Mással is így bánnál ebben a szituációban?",
+    "self_blaming_thought": "A fiam rosszalkodik az iskolában, biztos rosszul nevelem őt.",
+    "should_statements": "Hibás Biztos állítások",
```

Comment:
aren't these more like "Kell / muszáj állítások"... or "kellene, hogy" or something like that? (not sure how to translate though...) As in... for the airplane sentence, the faulty part is that "I shouldn't be afraid", not "surely the plane is okay" (... the plane is actually okay, but the fact that I shouldn't be afraid won't make me less afraid.)